packages/MarkdownEditor-Tests.package/MarkdownHeadingsTest.class/instance/testMismatchHeadingLevels1To6WithUnspacedText.st

```diff
@@ -1,6 +1,6 @@
 tests - matching
 testMismatchHeadingLevels1To6WithUnspacedText
 
-	#('#MiniMetro' '##the fastest game' '###on the planet!' '####cool play style'
-	'#####endless fun' '######universal gratitude of passengers') do: [:string | 
+	{'#hashtag'. '##missing space'. '###no heading'. '####just text'. '#####five hashes'.
```

Comment:
When you really want to delete the funny test strings then please be consistent. I know, the first ones are self explaining, but not the last ones.

Reply:
Okay, the last one `######long fenced railway gate` was meant as a compromise - as the reader should understand what's happening after the first five cases and then I don't care about the content.
Nevertheless, it was wrong before that the spaced and unspaced version use the same strings

Reply:
Yeah, that's an improvement, but the last element doesn't add anything to the test's semantic...